doc/conf.py

```diff
@@ -95,7 +114,7 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = ['.rst', '.md']
+source_suffix = ['.rst']
```

Comment:
I'm surprised that it still works after `.md` is removed 😱

Reply:
I didn't even mean to do that...

Reply:
But the `README` is included via `includemd` anyway

Reply:
Where is `includemd`?

Reply:
https://github.com/pygae/galgebra/blob/77320a5f8d0bc35c89c3d7eb81e6746193ca5049/doc/index.rst#L1